.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
```

Comment:
Is this for https://pypi.org/project/pre-commit/ ?

I would vote against having another dependency (I don't see it in poetry.lock though) so will it be an option for the user to install?